package.json

```diff
@@ -10,12 +10,10 @@
     "watch:drive:browser": "COZY_APP_SLUG=drive cozy-scripts watch --manifest src/drive/targets/manifest.webapp --build-dir build/drive --src-dir src/drive",
     "watch:drive:mobile": "COZY_APP_SLUG=drive NODE_ENV=mobile:development cozy-scripts watch --manifest src/drive/targets/manifest.webapp --src-dir src/drive",
     "watch:drive:standalone": "COZY_APP_SLUG=drive NODE_ENV=mobile:development cozy-scripts start --no-stack --manifest src/drive/targets/manifest.webapp --src-dir src/drive",
-    "build:photos": "npm-run-all --parallel build:photos:browser build:photos:services",
+    "build:photos": "npm-run-all --parallel build:photos:browser",
     "build:photos:browser": "COZY_APP_SLUG=photos:browser cozy-scripts build --manifest src/photos/targets/manifest.webapp --build-dir build/photos --src-dir src/photos",
```

Comment:
`COZY_APP_SLUG=photos:browser` should be replaced by `COZY_APP_SLUG=photos`, so `isPhotos` has the expected value